src/python/dxpy/bindings/dxfile_functions.py

```diff
@@ -95,7 +95,7 @@ def new_dxfile(mode=None, write_buffer_size=dxfile.DEFAULT_BUFFER_SIZE, expected
 
 
 def download_dxfile(dxid, filename, chunksize=dxfile.DEFAULT_BUFFER_SIZE, append=False, show_progress=False,
-                    project=None, **kwargs):
+                    project=None, describe_output=None, **kwargs):
```

Comment:
High level/clarification question.  For the new `describe_output` field in `download_dxfile`, for what code paths (if any) is the default value of None passed?

Reply:
Whenever `download_dxfile()` is called (since it's a part of our API) without the `describe_output` parameter.

Internally it's used 
* in `dx get` when resources are downloaded [here](https://github.com/dnanexus/dx-toolkit/blob/98b7f0ae600147cf7dcae7ab710ef38f07458e9a/src/python/dxpy/utils/executable_unbuilder.py#L183) 
* in the [`download_all_inputs`](https://github.com/dnanexus/dx-toolkit/blob/ef5d9a3dad811dd6dc7bfa6f34eb9a4a6669985d/src/python/dxpy/bindings/download_all_inputs.py#L49) script